packages/wp-story-editor/src/components/mediaUpload/mediaPicker/WordPressImageCropper.js

```diff
@@ -53,7 +53,7 @@ const WordPressImageCropper = window.wp?.media?.controller?.Cropper?.extend?.({
         : control.params.height;
     }
 
-    return wp.ajax.post('crop-image', {
+    return window.wp.ajax.post('crop-image', {
```

Comment:
🤔 Is this required or nice to have?

Reply:
Yes. Closing in favor of the [other open thread](https://github.com/GoogleForCreators/web-stories-wp/pull/10487#discussion_r800666513)